src/mrack/providers/openstack.py

```diff
@@ -432,26 +447,26 @@ async def wait_till_provisioned(
 
         start = datetime.now()
         timeout_time = start + timedelta(minutes=timeout)
-        done_states = ["ACTIVE", "ERROR"]
 
         # do not check the state immediately, it will take some time
         await asyncio.sleep(poll_sleep_initial)
 
-        error_attempts = 0
```

Comment:
This variable must be defined, it's used later inside the `while` block.

Reply:
my bad i moved that to the argument :D damn